pkg/api-server/dataplane_overview_endpoints.go

```diff
@@ -4,6 +4,8 @@ import (
 	"context"
 	"strings"
 
+	"github.com/Kong/kuma/pkg/api-server/types"
```

Comment:
We definitely should do something with `goimports`. I'll check maybe they had new releases

Reply:
fixed the import